dumpconverter.py

```diff
@@ -7,13 +7,16 @@
 from propertysuggester.utils.CompressedFileType import CompressedFileType
 
 if __name__ == "__main__":
-    logging.basicConfig(level=20) # Print logging.info
+    logging.basicConfig(level=20)  # Print logging.info
```

Comment:
Not that I care that much but just wondering: so this flake8 imposes the rule that a comment should be preceded by two space characters? Or why it got changed?

Reply:
Yes, before inline comment there should be at least two spaces (and one after the "#")

Reply:
https://www.python.org/dev/peps/pep-0008/#inline-comments (PEP8 is standard linting guide for python, flake8 uses it)

Reply:
I see. I didn't know that. Thanks!